modules/tinymce/src/themes/mobile/main/ts/util/SkinLoaded.ts

```diff
@@ -4,20 +4,19 @@
  * For LGPL see License.txt in the project root for license information.
  * For commercial licenses see https://www.tiny.cloud/
  */
+import Editor from 'tinymce/core/api/Editor';
```

Comment:
There's a missing line between the license header and content here.

- [ ] Fix missing blank line between header and imports